web/modules/custom/collection/collection.module

```diff
@@ -109,7 +109,7 @@ function collection_rdf_entity_access(EntityInterface $entity, $operation, Accou
 
     case 'edit':
       $allowed_transitions = $entity->field_ar_state->first()->getTransitions();
-      return AccessResult::forbiddenIf(empty($allowed_transitions));
+      return empty($allowed_transitions) ? AccessResult::forbidden() : AccessResult::allowed();
```

Comment:
This is not. Sorry. It is a hotfix that we had elsewhere but we did not backport for solutions. It was causing an access result failure on the site so I thought I could fix it right away since it's easy and we already test it elsewhere.